public/docs/ts/latest/guide/style-guide.jade

```diff
@@ -852,15 +852,15 @@ a(href="#toc") Back to top
 
 .s-rule.do
   :marked
-    **Do** be descriptive with file names and keeping the contents of the file to exactly one component.
+    **Do** be descriptive with file names and keep the contents of the file to exactly one component.
 
 .s-rule.avoid
   :marked
     **Avoid** files with multiple components, multiple services, or a mixture.
 
 .s-why.s-why-last
   :marked
-    **Why?** We spend less time hunting and pecking for code, and become more efficient. If this means we want longer file names, then so be it.
+    **Why?** We spend less time hunting and peeking for code, and become more efficient. If this means we want longer file names, then so be it.
```

Comment:
Make it ` **Why?** We spend less time hunting for code, and become more efficient. If this means we want longer file names, then so be it.`